videoframegrid.py

```diff
@@ -68,6 +71,28 @@ def swap_frame(self, frame):
             else:
                 self.selected_frame = frame
 
+    def toggle_fullscreen(self, selected_frame):
+        if not self.fullscreen:
+            for videoframe in self.videoframes:
```

Comment:
Ahh yeah nevermind. I did not think of running the commands inside the list comprehension, but rather of the if-statement before them. But it won't improve readability, so just ignore 📦